bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/dto/HeatingModel.java

```diff
@@ -153,14 +155,26 @@ public void setBatterylow(BigDecimal batterylow) {
         this.batterylow = batterylow;
     }
 
-    public BigDecimal getWindowopenactiv() {
+    public @Nullable BigDecimal getWindowopenactiv() {
```

Comment:
I know, it has not been introduced by you, but is it intentionally `getWindowopenactiv` and not `getWindowopenactive`?

Reply:
It has been introduced by me some years ago. I just reused the names of the XML tags for DTOs. And it is written in that way over there too.

Reply:
ok :-)